hashicorp/dev-portal
# feat: add nomad custom docs landing

## "Ready for Review" checklist




---

## Relevant links

- [Preview link][preview] 🔎
- [Asana task](https://app.asana.com/0/1201987349274776/1202129511371379/f) 🎟️

## What

Builds a new custom `/docs` landing page for `nomadproject.io`.

## Why

To improve the user experience of the `nomadproject.io/docs` landing page.

## How

Takes an identical approach to Vault in #178 & Consul in #264:

- Have split out `docs/[...page].tsx`, which is very similar to the former `[[...page]].tsx`
- New file `docs/index.tsx`, where the custom landing page is built

## Testing

To visit the preview product docs landing page, such as Consul's:

1. Open [the preview link][preview]
2. Select the desired product in the lower-left products dropdown. This switched the site's configuration to serve the product's dot-io site, rather than dev-portal.
3. Navigate to `/docs`

- [ ] Visit the Nomad docs landing page, and ensure the page looks and functions as expected
    - The new cards content should appear on the page
    - The sidebar should function as expected
    - Accessing versioned docs should function as expected. Note that the cards content is shown on the latest version only.

## Anything else?

Note that `nomadproject.io` has not yet been migrated to `dev-portal`.

[preview]: https://dev-portal-git-zsnomad-custom-docs-landing-hashicorp.vercel.app/docs
